nixos/modules/services/web-apps/invidious.nix

```diff
@@ -235,6 +304,16 @@ in
       '';
     };
 
+    address = lib.mkOption {
+      type = types.str;
+      # default from https://github.com/iv-org/invidious/blob/master/config/config.example.yml
+      default = if cfg.nginx.enable then "127.0.0.1" else "0.0.0.0";
```

Comment:
Either we do not open the firewall automatically, do we? Then this should be fine, I guess.